build.sbt

```diff
@@ -1,6 +1,6 @@
 scalaVersion := "2.12.4"
 
-name := "delphi"
+name := "delphi-cli"
```

Comment:
Did you intend to rename this?

Reply:
The executable generated will take this name. So I have renamed it. I can revert if it is a issue.

Reply:
Yes, that's why i actually renamed it. :)
This way the users can enter `delphi` and not `delphi-cli`. 
I think this ways its a bit stronger. What do you think?

Reply:
Ok then revert it to `delphi`. In documentation `delphi-cli` is used I renamed because of that https://github.com/delphi-hub/delphi-cli.